cupyx/scipy/sparse/data.py

```diff
@@ -211,7 +211,10 @@ def _arg_min_or_max_axis(self, axis, op, sum_duplicates):
         # Do the reudction
         value = mat._arg_minor_reduce(op, axis)
 
-        return value
+        if axis == 0:
```

Comment:
Note to myself: `_arg_min_or_max()` says `axis` is not `None` at this point, and currently we don't support negative axis until #3497, so it can only be 0 or 1 here.